sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -808,7 +1032,70 @@ the file extension is not supported or if the write operation failed.
 
 |===
 
-Table {counter: tableNumber}. Member functions of the `command_graph` class for queue recording.
+Table {counter: tableNumber}. Member functions of the `command_graph` class for
+graph update.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+void update(node& node);
+----
+
+| Updates an executable graph node that corresponds to `node`. `node` must be a
+kernel execution node. Argument values which have associated dynamic parameters
+will be updated, and the ND-range of the node will be updated inside the
+executable graph to reflect the current values in `node`.
+
+Preconditions:
```

Comment:
Yes I think you are correct, though there are a couple of usages in the[ SYCL spec (for example `kernel_bundle::get_kernel()` in this section)](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:bundles.query) that are the same as our usage here, they do seem to be outliers.

Since this will touch a lot of other areas of the spec I'll look at this as a separate future PR to avoid cluttering this one.

Reply:
> Yes I think you are correct, though there are a couple of usages in the[ SYCL spec (for example `kernel_bundle::get_kernel()` in this section)](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:bundles.query) that are the same as our usage here, they do seem to be outliers.

Yes, I noticed this too.  This is a bug in the spec that should be fixed.  I think I am the culprit for those references in the spec.  I wrote that section before I understood the difference between "precondition" and "constraint".

> Since this will touch a lot of other areas of the spec I'll look at this as a separate future PR to avoid cluttering this one.

Makes sense.